### DISCUSSION TITLE
Keypad Chirp control

### DISCUSSION THREAD

**Discussion by KewlGuyRox** [tsightler/ring-mqtt#797] [Feature Requests]
Is there a way to initiate a chirp on the keypad.. like how a piezo buzzer can be configured to beep? Typically the chirp sounds when the door is opened but not when the door is closed. So if someone leaves and closes the door, there is no audible note to confirm it.

**Comment by tsightler** [comment]
This is largely answered by the FAQ:

**Q) Is it possible to \<insert random thing not possible in the Ring app>**  
**A)** The goal of this project is to facilitate integration of devices in Ring's proprietary ecosystem with external systems via industry standard protocols to allow automation of functions that are available in the Ring app.  As this project uses the same API which is leveraged by Ring's own apps, if you can't perform a given function via the Ring app, it is unlikely to be possible with ring-mqtt.
